src/test/stemcontrolobjecttest.cpp

```diff
@@ -103,6 +103,7 @@ class StemControlTest : public BaseSignalPathTest {
         // CachingReaderWorker::trackLoaded (CachingReader) ->
         // EngineBuffer::loaded (CachingReader, direct) ->
         // BaseTrackPlayerImpl::slotTrackLoaded  (main)
+        QTest::qSleep(20); // sleep 20 ms for waiting event from CachingReaderWorker
```

Comment:
We already have a loop here that waits for
pEngineDeck->getEngineBuffer()->isTrackLoaded()
https://github.com/mixxxdj/mixxx/blob/437db42f5fc36ba8a1b6b0a6e84162d7525ba7d2/src/test/signalpathtest.h#L180

This means the event trackLoaded() should be already queued. 
https://github.com/mixxxdj/mixxx/blob/437db42f5fc36ba8a1b6b0a6e84162d7525ba7d2/src/engine/enginebuffer.cpp#L690

If this is not the case, something else seems to be wrong. 

At least the comments are missleading and should be adjusted.